lib/private/Files/Config/UserMountCache.php

```diff
@@ -383,6 +385,28 @@ public function getMountsForFileId($fileId, $user = null) {
 		}, $filteredMounts);
 	}
 
+	/**
+	 * Get all nodes that give access to a file
+	 *
+	 * @return array<string, Node[]> Nodes giving access to the given fileId, indexed by user
```

Comment:
```suggestion
	 * @return array<string, Node[]> Nodes giving access to the given fileId, indexed by user ID
```

just a suggestion for clarity